packages/server/src/__tests__/integration/apolloServerTests.ts

```diff
@@ -1772,40 +1771,6 @@ export function defineIntegrationTestSuiteApolloServerTests(
         expect(result.errors[0].extensions.exception).toBeDefined();
         expect(result.errors[0].extensions.exception.stacktrace).toBeDefined();
       });
-
-      it('shows ApolloError extensions in extensions (only!)', async () => {
```

Comment:
This might still be a valuable test if you change it to use something like AuthenticationError? Unless that already exists?

Reply:
Yes, it's valuable.
I added it back as a separate commit